pkg/upgrades/v0_6_0.go

```diff
@@ -0,0 +1,57 @@
+package upgrades
```

Comment:
We have recently released 0.6.0. So this belongs to the next release, supposedly, `v0.7.0`